AUTHORS

```diff
@@ -999,12 +999,59 @@ Shubham Agrawal <shubham.ag6845@gmail.com>
 numbermaniac <5206120+numbermaniac@users.noreply.github.com>
 Sakirul Alam <binarysakir@gmail.com>
 Mohammed Bilal <r.mohammedbilal@gmail.com>
+Naveen Sai <naveensaisreenivas@gmail.com>
 Chris du Plessis <christopherjonduplessis@gmail.com>
 Coder-RG <rgoel1999@gmail.com>
 Ansh Mishra <anshmishra471@gmail.com>
 Alex Malins <github@alexmalins.com>
 Lorenzo Contento <lorenzo.contento@gmail.com>
-Naveen Sai <naveensaisreenivas@gmail.com>
 Shital Mule <shitalmule04@gmail.com>
 Amanda Dsouza <meezamanda@yahoo.com>
 Nijso Beishuizen <nijso@koolmees.numerically-related.com>
+Harry Zheng <harry@harryzheng.com>
+Felix Yan <felixonmars@archlinux.org>
+Constantin Mateescu <costica1234@me.com>
+Eva <eva.tiwari@gmail.com>
+aditya113141 <adityakumar113141@gmail.com>
+Soumi Bardhan <51290447+Soumi7@users.noreply.github.com>
+Kaustubh <ckaustubhm06@gmail.com>
+Kristianmitk <hello@kristianbrunn.com>
+Neel Gorasiya <mgorasiya1974@gmail.com>
+Akshat Sood <68052998+akshatsood2249@users.noreply.github.com>
+jmgc <chemoki@gmail.com>
+Stefan Petrea <stefan@garage-coding.com>
+povinsahu1909 <povinsahu@gmail.com>
+Mark Bell <mark00bell@googlemail.com>
+AlexCQY <alex_chua@u.nus.edu>
+FFroehlich <fabian@schaluck.com>
+gopalamn <gopalamn@umich.edu>
+Kartik Sethi <kartiks31416@gmail.com>
+MaqOwais <quadirowais200@gmail.com>
+Harshit Yadav <harshityadav2k@gmail.com>
+Sidharth Mundhra <sidharthmundhra16@gmail.com>
+suryam35 <suryamkalra35@gmail.com>
+LAPTOP-AS1M2R8B\codem <codemastercpp@gmail.com>
+kunal singh <ksingh19136@gmail.com>
+mayank raj <mayank_1901cs35@iitp.ac.in>
+mayank <54908605+mayankray2020@users.noreply.github.com>
+JSS95 <52185322+JSS95@users.noreply.github.com>
```

Comment:
@jss95 should I change this? Are you already in the AUTHORS file under a different name?

Reply:
I see `Jisoo Song` as well. Should I alias them all to that?

Reply:
I've aliased them to that.